### PR TITLE
Update eip-2535.md

### PR DESCRIPTION
**Pull request by mudgen**

### CODE REVIEW

**Comment by mudgen** [comment]
@MicahZoltu I think this pull request got stuck.  Do you know how I can fix it or who I should ask for help about it?

**Comment by MicahZoltu** [comment]
CI has been acting up, and I don't know how to fix it.  Maybe @lightclient has some ideas?  Though I think I saw them in another thread mention that they were also stumped.  😢

File: EIPS/eip-2535.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ New terminology from the diamond industry.
...
-4. Solves the 24KB maximum contract size limitation. Diamonds can be any size.
-5. Enables zero, partial or full diamond immutability as desired, and when desired.
-6. Designed for tooling and user-interface software.
+1. A way to create a set of contracts (facets) under a single address that are independent from each other but can share functionality and contract storage.
+2. A way to add, replace and remove multiple external functions atomically (in the same transaction).
+3. An event that shows what functions are added, replaced and removed.
+4. A way to look at a diamond to see its facets and functions.
+5. Solves the 24KB maximum contract size limitation. Diamonds can be any size.
+6. Enables zero, partial or full diamond immutability as desired, and when desired.
+7. Designed for tooling and user-interface software.
```
Comment (MicahZoltu) [inline EIPS/eip-2535.md:33]: You can just use the number `1` for each line item so you don't have to maintain the numbers manually.
```
1. First.
1. Second.
1. Third.
```
> 1. First.
> 1. Second.
> 1. Third.

**Comment by lightclient** [comment]
Yeah, unfortunately I am not sure what the cause is. I see that Github was having [some issues](https://www.githubstatus.com/incidents/21jl0kww3xfc) with actions recently, but it seems like we've been having issues on this repo for a few days now. The thing is, the jobs are [still running](https://travis-ci.org/github/ethereum/EIPs/builds/718813420) in Travis CI. The Github UI just isn't aware of the results for some reason.

File: EIPS/eip-2535.md (hunk 1 above)
Comment (mudgen) [reply EIPS/eip-2535.md:33]: Wow, that is a great tip!  Thanks @MicahZoltu

**Comment by axic** [comment]
Travis always had intermittent issues, nothing new here. In order to retrigger Github, one must push a different commit hash. Closing/opening won't help. Just push a new commit, rebase it on top of master, or merge master in. Anything which results in a new commithash.

**Comment by mudgen** [comment]
@axic Thanks!  How long does it take when it works?

**Comment by mudgen** [comment]
@MicahZoltu @lightclient I'm really eager for this pull request to be merged.  Can one of you please merge it for me?

**Comment by lightclient** [comment]
We need to figure out what's going on with CI, but AFAICT the tests are passing: https://travis-ci.org/github/ethereum/EIPs/builds/719329719

I would say let's go ahead and merge, but I'm worried if there will be any implications w.r.t. deploying the `eips.ethereum.org` website.

**Comment by mudgen** [comment]
@lightclient Great, I can let you know if the changes show up on `eips.ethereum.org`.

**Comment by MicahZoltu** [comment]
No one can merge without CI passing except admins, and the admins are also the ones who need to look at CI.

**Comment by mudgen** [comment]
@MicahZoltu Who are the admins?

**Comment by mudgen** [comment]
@MicahZoltu @axic @lightclient I need this merged.  I did add another commit.  Who can I talk to get this working or merged?   Is this something Github support can help with?  Can I ask them?

**Comment by lightclient** [comment]
@mudgen we are currently investigating the issue. Until we resolve the issue with the CI, we won't be able to get this merged. You by all means welcome to contact Github / Travis support in the mean time. Please share any updates with us here.

**Comment by mudgen** [comment]
@lightclient I understand. Much appreciated.

**Comment by mudgen** [comment]
I made another, different pull request hoping a different one might go through.  So I'm closing this pull request.  The new pull request is here: https://github.com/ethereum/EIPs/pull/2886

**Comment by MicahZoltu** [comment]
Sadly, CI is broken repository wide right now, not just on this PR.  😢

**Pull request closed without merging**